custom_components/miele/translations/de.json

```diff
@@ -417,10 +416,10 @@
                             "dust_box_missing": "Fehlender Staubbehälter",
                             "blocked_drive_wheels": "Antriebsrad blockiert",
```

Comment:
<strike>Should be `Antriebsräder blockiert`, judging from the English version, but I don't know enough about the robot's hardware.</strike> 

According to the error message description in the Scout RX3 Home Vision manual: `Überlastung Antriebsräder`